examples/with-realm-web/pages/index.js

```diff
@@ -0,0 +1,84 @@
+import useSWR from 'swr'
+import { generateAuthHeader, REALM_GRAPHQL_ENDPOINT } from '../lib/RealmClient'
+import { FIND_MOVIES } from '../lib/graphql-operations'
+
+const fetcher = async (url) =>
+  fetch(url, {
+    method: 'POST',
+    headers: await generateAuthHeader(),
+    body: JSON.stringify({
+      query: FIND_MOVIES,
+    }),
+  }).then((res) => res.json())
+
+const IndexPage = () => {
+  const { data, error } = useSWR(REALM_GRAPHQL_ENDPOINT, fetcher)
+
+  const movies = data ? data.data.movies : null
+
+  if (error) {
+    console.log(error.message)
+  }
```

Comment:
Handle the error, return a paragraph with the error message or something like that.

Reply:
I'm doing this now 
```
console.error(data.error)
return `An Error occurred while trying to fetch data from the realm endpoint :  ${data.error}`
```

should I 'throw' and error so it uses next.js error overlay?

Reply:
Nope, that's invalid JSX I think, just return a paragraph with the error message, something like:

```jsx
return <p>An error happened: ${error.message}</p>
```

Reply:
Ah! Alright. Will update in a sec

Reply:
Done! Have a look @lfades